core/lib/spree/core/number_generator.rb

```diff
@@ -0,0 +1,46 @@
+module Spree
+  module Core
+    class NumberGenerator < Module
+
+      RAND           = Random.new
+      BASE           = 10
+      DEFAULT_LENGTH = 9
+      NUMBERS        = (0..9).to_a.freeze
+      LETTERS        = ('A'..'Z').to_a.freeze
```

Comment:
@JDutil I'd consider to change the rubocop config to allow this kind of alignment. It makes the code much easier to read and understand. If you agree I'd PR a change to rubocop config.